### PR TITLE
Fix migration failure by removing legacy database class

### PR DESCRIPTION
**Pull request by andreheeke**
PR from #24

### CODE REVIEW

File: src/Migration/AutogenerateBookingForm.php [hunk 1]
```diff
@@ -83,22 +90,31 @@ public function run(): MigrationResult
      */
     private function autoGenerateBookingForm(): void
     {
+        // Initialize the contao framework
+        $this->framework->initialize();
+
```
Comment (markocupic) [inline src/Migration/AutogenerateBookingForm.php:95]: Why do you Initiative the Contao framework herein?

Comment (andreheeke) [reply src/Migration/AutogenerateBookingForm.php:95]: In line #L121 you have to call the method to load a data container array. As far as I know, the new migrations run independently from the Contao system, so you have to initialize the system first.

Comment (markocupic) [reply src/Migration/AutogenerateBookingForm.php:95]: Ah.... clearly. But then we should move this call down to line 119, shouldn't we?

Comment (andreheeke) [reply src/Migration/AutogenerateBookingForm.php:95]: Yes, we can. ;) I've seen that you have already changed this order.

**Pull request merged**